numpy/lib/type_check.py

```diff
@@ -541,6 +541,9 @@ def real_if_close(a, tol=100):
 
 
 def _asscalar_dispatcher(a):
+    # 2018-10-10, 1.16
+    warnings.warn('np.asscalar(a) is deprecated since NumPy v1.16, use '
+                  'a.item() instead', DeprecationWarning, stacklevel=2)
```

Comment:
```suggestion
                  'a.item() instead', DeprecationWarning, stacklevel=3)
```
This should have been level 2 to begin with.